src/main/java/bio/terra/app/configuration/BigQueryConfiguration.java

```diff
@@ -1,4 +1,4 @@
-package bio.terra.service.filedata.google.bq;
+package bio.terra.app.configuration;
```

Comment:
Is this re-org one that the team has agreed to? I think there are benefits to keeping a configuration class in the same package as the classes it's handling configuration for. I don't see a great vault in putting all the configuration classes in the same package.

Reply:
We currently do this for quite a few configuration classes, so I was just following that trend. I can put it back, but we should decide on the way we're going to do things.

Reply:
I don't feel super strongly about it either way but I'm wondering if there's an argument in favor of collecting them in a single package, other than they're all configuration classes. I can't think of one but there's probably one I haven't thought of...

Reply:
Since all the configurations are confined to one `application.properties` file, it makes sense to group all the code that references that file into one package, even if they're providing properties to different services. It becomes more akin to a configuration "layer".